emu/src/bitwise.rs

```diff
@@ -220,12 +264,37 @@ mod tests {
 
     #[test]
     fn set_bit() {
-        let mut b = 0b1100110_u32;
+        let mut b = 0b10001001_u32;
         b.set_bit(0, true);
         b.set_bit(1, true);
         b.set_bit(2, false);
         b.set_bit(3, false);
-        assert_eq!(b, 0b1100011)
```

Comment:
I think we can revert this change right?

Reply:
Yeah I kind of agree on this, I really only use `set_bits` testing for testing purposes I find it handy, but i can change this quite easily

Reply:
Sorry what I meant is: why are we changing the test of `set_bit`? We're not touching it in this PR no?

Reply:
Oh ok, I have no idea how that changed :joy: , I'll revert it np